ubuntu/get-docker.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/sh
+
+
+if ( apt-cache show ca-certificates )
+then
+  echo "-> ca-certificates is in packages list"
+else
+  echo "-> cannot find ca-certificates, updating package list"
+  sudo DEBIAN_FRONTEND=noninteractive apt update
+fi 
+
+if ( apt-cache show curl )
+then
+  echo "-> curl is in packages list"
+else
+  echo "-> cannot find curl, updating packages list"
+  sudo DEBIAN_FRONTEND=noninteractive apt update
+fi
+
+if ( apt-cache show gnupg )
+then
+  echo "-> gnupg is in packages list"
+else
+  echo "-> cannot find gnupg, updating packages list"
+  sudo DEBIAN_FRONTEND=noninteractive apt update
+fi
+
+if [ -f /etc/ssl/certs/ca-certificates.crt ]
+then
+  echo "-> /etc/ssl/certs/ca-certificates.crt exists"
+else
+  echo "-> installing ca-certificates"
+  sudo apt install ca-certificates
+fi
+
+if ( which curl )
+then
+  echo "-> curl is installed"
+else
+  echo "-> installing curl"
+  sudo apt install curl
+fi
+
+if ( which gpg )
+then
+  echo "-> gnupg is installed"
+else
+  echo "-> installing gnupg"
+  sudo apt install gnupg
+fi
+
+if [ -d /etc/apt/keyrings ]
+then
+  echo "-> /etc/apt/keyrings exists"
+else
+  echo "-> creating /etc/apt/keyrings"
+  sudo install -m 0755 -d /etc/apt/keyrings
```

Comment:
Using `install` over `mkdir`?

Reply:
honestly I was just following what the documentation to install on ubuntu says.
https://docs.docker.com/engine/install/ubuntu/
scroll down to "Install using apt repository" but i changed it to `mkdir -m 0755 -p <path>`